app/controllers/import_users_controller.rb

```diff
@@ -56,13 +56,6 @@ def valid_input_file?(imported_users, file) # rubocop:disable all
       redirect_to(:back) && return
     end
 
-    # make sure that there's spaces in the header line
```

Comment:
Just something to note - this change was originally only made for *User* CSV import, but we also have equipment import that shouldn't be changed. Since they both use the `CsvImport` library, the changes in this PR will extend these to both imports.